.github/CODEOWNERS

```diff
@@ -279,7 +273,6 @@ azure-pipelines.yml @coq/ci-maintainers
 # Secondary maintainer    @silene
 
 /tools/coqdep*          @ppedrot
-# Secondary maintainer @maximedenes
```

Comment:
I suppose coqdep should be maintained by the build system team (doesn't exist yet, but it seems time to create it).

Reply:
I can help with `coqdep`, indeed that should be under the build-team.

The issue exists tho [just pointing out not willing to start a debate here] , I have a largish set of patches for `coqdep` , then it is in my interested _not_ to be a maintainer. In case a) [I'm not a maintainer] theoretically Pierre-Marie and Maxime could assign and merge my patchset quickly, however, if I am also a maintainer, only one of them could, thus that would reduce the bandwidth by half.

Reply:
>The issue exists tho [just pointing out not willing to start a debate here] , I have a largish set of patches for coqdep , then it is in my interested not to be a maintainer. In case a) [I'm not a maintainer] theoretically Pierre-Marie and Maxime could assign and merge my patchset quickly, however, if I am also a maintainer, only one of them could, thus that would reduce the bandwidth by half.

I think Maxime is leaving maintainership of coqdep in this PR though
And when using a maintainer team we're not limited to 2 maintainers, so being one doesn't mean only 1 person can merge your patches

Reply:
Indeed maybe teams are solution.

Reply:
Apart from @coq/ci-maintainers, all the other maintainer teams have at least three people in them (and I could actually join @coq/ci-maintainers if you needed, but you two have been pretty efficient so far). The build system team would likely include @ejgallego, @gares, @SkySkimmer, and myself. The only issue is that @ppedrot is likely not willing to be in this team even if he's the current principal owner of coqdep.